src/commands.ts

```diff
@@ -475,19 +578,25 @@ export class Commands {
 			);
 			return;
 		}
-
-		if (asset.nft.owner !== (await this.signer.getAddress())) {
+		const assetOwner = asset.nft.owner
+		let serviceType
+		if (isVerifiableCredential(asset)) {
+			serviceType = (asset as any).credentialSubject.services[0].type
```

Comment:
IMO, we're doing this on the wrong way
we're pretty much force setting all the types to be "any" because it relies on ` ocean.js `updates that were not done yet,  like the Property '`credentialSubject`' on 'Asset' for instance. It should be the other way around... FYI, we're introducing other ocean.js changes targeting next version branch (will be 4.0 with eventually some breaking changes) not main branch..
In any case, i think we should have an updated version of ocean.js before introducing these changes on the CLI and the node

Reply:
ok, thanks, i agree